### PR TITLE
User Guide: Fix date format in example

### LINKED ISSUE #198: [PE-D][Tester A] UG has inconsistent date format for command 'CREATE_EVENT'
Acceptable values is said to be in the form of YYYY-MM-DD, but example given is DD-MM-YYYY. Actual product follows YYYY-MM-DD, so may need to change example.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: EricXiong420/ped#1

### PR DESCRIPTION
**Pull request by andrechuakj**
Closes #198 

What have you done in this PR?
* Updated createEvent example to have the correct date format
* Updated createEvent command format `[tag]` to have ellipsis

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -107,12 +107,12 @@ Examples:
...
 Creates a new event with accompanying details (name, location, date, tag).
 
-Format: `createEvent n/EVENT_NAME l/LOCATION d/DATE [t/TAG]`
+Format: `createEvent n/EVENT_NAME l/LOCATION d/DATE [t/TAG]...`
 
 * Acceptable values for `DATE`: Dates in the format of `YYYY-MM-DD`.
 
 Examples:
-* `createEvent n/Party l/Raffles Hall d/16-09-2023` creates an event `Party` in CCACommander.
+* `createEvent n/Party l/Raffles Hall d/2023-09-16` creates an event `Party` in CCACommander.
```
Comment (SelwynAng) [inline docs/UserGuide.md:115]: We should update the date format in the Command Summary section to YYYY-MM-DD too. Currently it is still in the DD-MM-YYYY format.

**Review by SelwynAng — CHANGES_REQUESTED** [review]
Command Summary section needs to be updated.

File: docs/UserGuide.md (hunk 1 above)
Comment (andrechuakj) [reply docs/UserGuide.md:115]: Updated

**Review by SelwynAng — APPROVED** [review]
LGTM!

**Pull request merged**